invites_test.go

```diff
@@ -0,0 +1,37 @@
+package rollbar
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestListInvites(t *testing.T) {
+	teardown := setup()
+	defer teardown()
+	handUrl := fmt.Sprintf("/team/%s/invites/", teamID)
+
+	mux.HandleFunc(handUrl, func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+
+		// Check if we've hit the second page and return an empty paginated
+		// response thus simulating the apis behaviour.
+		splStr := strings.Split(r.URL.RawQuery, "&")
+		if splStr[1] == "page=2" {
```

Comment:
Reverse if and else cases, make make anything not  `page=0` or `page=1` default behaviour and else cases do `fmt.Fprint(w, fixture("teams/empty_invites.json"))`. This is more intuitive and more correct because this is how the API would behave for calls with page > 1.

Moreover, you can just access tue query parameter's value:

```
  page  := r.URL.Query().Get("pages")
  fixtureName := if page == "0" || page == "1" {
    "invites"
  } else {
   "empty_invites"
  }
  fmt.Fprint(w, fixture(`teams/${fixtureName}.json`))
```

([Documentation](https://golang.org/pkg/net/url/#Values.Get))